lib/cartopy/crs.py

```diff
@@ -1140,6 +1140,66 @@ def y_limits(self):
         return self._y_limits
 
 
+class LambertAzimuthalEqualArea(Projection):
+    """
+    A Lambert Azimuthal Equal-Area projection.
+
+    """
+
+    def __init__(self, central_longitude=0.0, central_latitude=0.0,
+                 false_easting=0.0, false_northing=0.0,
+                 globe=None):
+        """
+        Kwargs:
+
+            * central_longitude - The central longitude. Defaults to 0
+            * central_latitude - The central latitude. Defaults to 0.
+            * false_easting - X offset from planar origin in metres.
+                              Defaults to 0.
+            * false_northing - Y offset from planar origin in metres.
+                               Defaults to 0.
+            * globe - A :class:`cartopy.crs.Globe`.
+                      If omitted, a default globe is created.
+
+        """
+        proj4_params = [('proj', 'laea'),
+                        ('lon_0', central_longitude),
+                        ('lat_0', central_latitude),
+                        ('x_0', false_easting),
+                        ('y_0', false_northing)]
+
+        super(LambertAzimuthalEqualArea, self).__init__(proj4_params,
+                                                        globe=globe)
+
+        a = np.float(self.globe.semimajor_axis or 6378137.0)
+        b = np.float(self.globe.semiminor_axis or 6356752.3142)
```

Comment:
I know this isn't the first PR to introduce these two radius values (6378137.0 and 6356752.3142), but please could you factor them out as constants: e.g. `WGS84_SEMIMAJOR_AXIS` and  `WGS84_SEMIMINOR_AXIS`.